src/layouts/EditPage/EditPageLayout.tsx

```diff
@@ -37,16 +35,17 @@ import { sanitiseRawHtml, updateHtmlWithMediaData } from "./utils"
 interface EditPageLayoutProps {
   getEditorContent: () => string
   setEditorContent: (content: string) => void
+  shouldDisableSave?: boolean
   variant: PageVariant
 }
 
 export const EditPageLayout = ({
   getEditorContent,
   setEditorContent,
   variant = "markdown",
+  shouldDisableSave = false,
```

Comment:
nit: rename to `isAnyDrawerOpen`. 
there are other reasons why save should be disabled (eg js content in the editor), so save button being disabled does not necessarily mean the drawer is open.

Reply:
not doing this change - this isn't meant to be a specific "is this drawer open" prop. this is more of "is my parent component telling me to disable save" of which a possible reason could be that the drawer is open. the context is given _in the component itself_, where we say that whether we disable save is based on whether the drawer is open.

hope this makes sense!